javascript/computer_science/project_hash_map.md

```diff
@@ -69,6 +71,43 @@ if (index < 0 || index >= buckets.length) {
 
 Remember that a hash map does not preserve insertion order when you are retrieving your hash map's data. It is normal and expected for keys and values to appear out of the order you inserted them in.
 
+#### Test Your Hash Map
+
+1. Create a new JavaScript file.
+
+1. Create a new instance of your hash map and set the load factor to be `0.75`.
```

Comment:
How about adding some code below this like:
```suggestion
1. Create a new instance of your hash map and set the load factor to be `0.75`.

    ```javascript
    const test = new HashMap() // or HashMap() if using a factory
    ```
```
Something like this would just make it more apparent what the `test` object is that you're calling `set()` on in the next step.

Reply:
Oops, you're right! I will add this in. I didn't realise this. Thanks!

And for the Ruby section, I'm more than happy to make the same changes to that as well and will raise a separate PR for that.

Thanks for the feedback!